src/input.h

```diff
@@ -181,6 +181,12 @@ class input_manager
         const std::vector<input_event> &get_input_for_action( const std::string &action_descriptor,
                 const std::string context = "default", bool *overwrites_default = NULL );
 
+        /**
+         * Return first char associated with an action ID in a given context.
+         */
+        char get_first_char_for_action( const std::string &action_descriptor,
+                                        const std::string context = "default" );
```

Comment:
Why is there a default value for the second parameter? All calls to this function supply a specific value anyway, the default is *never* used. And why is it a const value, but the first parameter is a reference to const?

Reply:
That was copied from `get_input_for_action`.